### ISSUE TITLE
Exposure check failed when opening the CWA Version 1.6.0 / 1.6.1 (ExposureDetectionIsAlreadyRunning) 

### ISSUE THREAD

**Issue by tibor** [corona-warn-app/cwa-app-ios#1497]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug

When opening the app, I got an error message „Exposure check failed“ and „An error occured during exposure logging. Code: ExposureDetectionIsAlreadyRunning“.

## Expected behaviour

It should work as normally with former versions of the app.

## Steps to reproduce the issue

![IMG_0161](https://user-images.githubusercontent.com/3136416/98805926-a5898400-2418-11eb-9deb-99ddc12b2072.PNG)
![IMG_0160](https://user-images.githubusercontent.com/3136416/98805931-a8847480-2418-11eb-814e-4225fe29a86d.PNG)

## Technical details

- iOS Version: 14.2
- Device: iPhone 11 Pro
- App Version: 1.6.0 (4)

---
Internal Tracking ID: EXPOSUREAPP-3683

**Comment by Ein-Tim** [comment]
@tibor Is This Issue new in 1.6.0?

**Comment by tibor** [comment]
@Ein-Tim haven’t seen this behavior in any of the previous app versions, so at least for me it’s new.

**Comment by thomasaugsten** [comment]
Did you restart the app?

**Comment by dsarkar** [comment]
Dear @tibor, dear community,

Thanks for posting this issue.

After updating to Version 1.6.0 it can happen, that you get a notification "Exposure check failed", although the last check occurred less than 24 hours ago. Currently the developers are working on a solution of this issue. Until the update is available, you can resolve this issue by restarting the App and if necessary restarting the phone.

Nach einem Update auf Version 1.6.0 kann es vorkommen, dass Sie die Nachricht "Risiko-Überprüfung fehlgeschlagen" erhalten, obwohl die letzte Überprüfung noch keine 24 Stunden her ist. Wir arbeiten gerade an einer Behebung dieses Problems. Bis das Update zur Verfügung steht, können Sie das Problem in den meisten Fällen mit einem Neustart Ihrer App beheben.

Best wishes,
DS

Corona-Warn-App Open Source Team

**Comment by ferreirafabio** [comment]
I have the same error and the same setup as @tibor

**Comment by thomasaugsten** [comment]
Can you please try v1.6.1

**Comment by tibor** [comment]
@thomasaugsten works well so far. Thanks for the fast fix 🚀

**Comment by ferreirafabio** [comment]
Works well for me again now too. Appreciate the fast responses.

**Comment by tibor** [comment]
@thomasaugsten unfortunately, I got the same error again today in the morning with 1.6.1
![IMG_0180](https://user-images.githubusercontent.com/3136416/98922311-98c96680-24d2-11eb-9f20-862a4b6e601c.jpg)
![IMG_0179](https://user-images.githubusercontent.com/3136416/98922324-9b2bc080-24d2-11eb-870d-e50a8e05080a.jpg)

**Comment by tibor** [comment]
Updated date == time when I opened the app

**Comment by dsarkar** [comment]
@tibor, each time when you open CWA version 1.6.1 it does an update, or so far only once in the morning? Can you close the up (swipe up to fully close) and reopen, please? Still time 8:38 or actual time? Thanks.

**Comment by tibor** [comment]
@dsarkar still 8:38, so the app updated the first time I opened the app.

**Comment by dsarkar** [comment]
@tibor thanks. 
1. So only one update of exposure logs per day probably. Let's observe for the next few days if you get more than one update per day.
2. The error message on your screenshot appeared when you updated to v1.6.1 and opened for the first time the updated CWA or did it appear sometime later?

**Comment by thomasaugsten** [comment]
@tibor You can ignore the alreadyRunning error this can happen when two exposure check started at almost the same time. We will improve the message in the future.

**Comment by Ein-Tim** [comment]
I can't reproduce [this](https://github.com/corona-warn-app/cwa-app-ios/issues/1497#issuecomment-725960568) Error with 1.6.1 @tibor

**Comment by tibor** [comment]
@thomasaugsten Thanks, I’m fine with that. Just wanted to let you know that the behavior is not (completely) fixed with 1.6.1

**Comment by Ein-Tim** [comment]
@tibor There is now a PR which fixes the Error (https://github.com/corona-warn-app/cwa-app-ios/pull/1510)

**Comment by tibor** [comment]
@Ein-Tim great, thanks!

**Comment by Ein-Tim** [comment]
@tibor Maybe you could change the title of this Issue to "Exposure check failed when opening the CWA Version 1.6.0 / 1.6.1 (ExposureDetectionIsAlreadyRunning) to avoid duplicates? 
Thanks

**Comment by FrankBuchholz** [comment]
I get the same error messages on
CWA 1.6.1
iPhone 8 with iOS 14.2

**Comment by Ein-Tim** [comment]
@FrankBuchholz This will be solved with the upcoming 1.7.0 release next week

**Comment by thomasaugsten** [comment]
@tibor Can you please update manually on v1.7.1 and check again if this problem is fixed

**Comment by Ein-Tim** [comment]
I can confirm that this Issue is not here anymore under 1.7.1 and the App also don't show [this](https://user-images.githubusercontent.com/3136416/98805926-a5898400-2418-11eb-9deb-99ddc12b2072.PNG) screen anymore 👍

**Comment by dsarkar** [comment]
Dear community,

We would appreciate very much detailed feedback on this issue, now that CWA version 1.7 has been rolled out. Please monitor the behavior over the next few days. Thank you all very much for your contributions. 

Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
I did not experience this Issue anymore under 1.7.1 👍
@tibor What about you?

**Comment by dsarkar** [comment]
@tibor, any feedback would be very much appreciated. Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by tibor** [comment]
@dsarkar so far, for the last days, I haven't seen the exposure check failed message no more. Seems like it's working fine now 🙂

**Comment by dsarkar** [comment]
Dear @tibor, dear community, thanks for the feedback! We close this issue now. Please re-open in case of new occurrences! 

Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@dsarkar Could you maybe remove the "Hot Topic" Lable so [this](https://github.com/issues?q=is%3Aissue+label%3A%22hot+topic+%F0%9F%94%A5%22+) overview over all the Hot Topic Issues stays clean and only active Issues are shown there? Thanks!
